continuedev/continue
# feat: add free trial card to onboarding

## Description

Adds a free trial card to onboarding, and combines the "Best" and "Custom" steps into a single card. Also creates a `FREE_TRIAL_LIMIT_REQUESTS` and `hasPassedFTL()` for common logic.

## Checklist


## Testing
- `rm ~/.continue/config.json`
- Go through onboarding, select "Best Experience", go through "Best models"
- Confirm config.json is updated correctly
- Repeat these steps for the "custom" config, and for the free trial by logging into GH

## Screenshots

### Onboarding home
<img width="466" alt="Screenshot 2024-06-27 at 5 17 53 PM"  src="https://github.com/continuedev/continue/assets/20157849/08a28de7-b621-4992-98ef-2d7aada42e3e">

### API key config
<img width="463" alt="Screenshot 2024-06-27 at 5 36 47 PM" src="https://github.com/continuedev/continue/assets/20157849/365aa251-bfcb-4a72-b861-88bb9afacf57">
